### ISSUE TITLE
create an external page of available command documentation

### ISSUE THREAD

**Issue by dasbif** [JoshCode/gilmore#26]
Similar to the following page for Xanbot on Twitch: http://xanbot.3v.fi/help.docs/

Create a linkable page (to add to !about and/or !help) with documentation of all commands, their arguments, their permission levels, and a brief explanation of their uses.

People don't call commands if they don't know they exist.  Also, users like me (who will call commands often) like having a list at hand so they don't have to spam a few !help !list type commands in chat.

**TL;DR having a static link to commands documentation is a useful feature.**
